sources/dataset.py

```diff
@@ -426,16 +430,19 @@ def draw_image(self, image_id, datapath):
 
         image = np.ones([self.image_size, self.image_size, 3], dtype=np.uint8)
         image = image * np.array(image_info["background"], dtype=np.uint8)
+        labelled_image = np.ones([self.image_size, self.image_size, 3], dtype=np.uint8) * 255
 
         # Get the center x, y and the size s
         if image_info["labels"][0]:
```

Comment:
Just a question: what is the value returned by `image_info["labels"][0]`? Is it a boolean, a digit (0 or 1)? I will be more explicit if you can read something like `if image_info["labels"]["type"] == "circle"`. In this way, you know the type of the geometry shape.

Reply:
`image_info['labels']` is a dictionary such that each label id is the key of a 0/1 value which indicates whether or not there is the corresponding shapes on the picture.
For instance:
```
image_info['labels'] = {'0': 0, '1': 1, '2': 1}
```
In such a case, there are one object of type '1' (a circle) and one object of type '2' (a triangle), but no object of type '0' (a square).
As far as I understand your point, we can not use it as `image_info['labels']['type']`, as it may exist several shapes on the same image. It is similar with the Mapillary dataset, where several objects may exist on the same street-scene image. However, couldn't we consider that 0/1 integer behave just like boolean values?

Reply:
Class attributes have been added as follows:
```
SQUARE = 0
CIRCLE = 1
TRIANGLE = 2
```
We can use them as in `image_info['labels'][self.SQUARE]` (for instance) if needed.